app/controllers/catalog_controller.rb

```diff
@@ -295,6 +295,7 @@ class CatalogController < ApplicationController
     # ===========================
 
     # Collection Show Page - Summary Section
+    config.add_summary_field "collectionnumber", field: "collection_number_ssi", helper_method: :render_html_tags
```

Comment:
Nit: you probably don't need render_html_tags helper as should be no html in the content